### PR TITLE
hrsc2isis support for level 3 images

### LINKED ISSUE #5497: HRSC level 3 images
**ISIS version(s) affected**: latest

**Description**  

The [hrsc2isis](https://isis.astrogeology.usgs.gov/8.1.0/Application/presentation/Tabbed/hrsc2isis/hrsc2isis.html) program does not support level 3 images.

But level 2 images are no longer in PDS, it appears. At the link https://pds-geosciences.wustl.edu/mex/mex-m-hrsc-3-rdr-v4/ there are only new images that are level 3. Previous versions have been archived, per their release notes.

I was able to use  hrsc2isis with level 3 by simply changing 3 to 2 in the .img file. 

Maybe this tool can be revisited to accept level 3. Here's a recipe that shows the current workaround: https://stereopipeline.readthedocs.io/en/latest/examples/hrsc.html

### PR DESCRIPTION
**Pull request by amystamile-usgs**
## Related Issue

closes #5497 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Infrastructure change (changes to things like CI or the build system that do not impact users)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added my user impacting change to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document. 

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by amystamile-usgs** [comment]
Looking into `Hrsc2isis.Hrsc2IsisTestProjection` test failure.

File: isis/src/mex/apps/hrsc2isis/hrsc2isis.cpp [hunk 1]
```diff
@@ -69,15 +69,6 @@ namespace Isis{
       throw IException(IException::User, msg, _FILEINFO_);
     }
 
-    // This program is setup to work with Mex HRSC processing level 1 and 2 only.
-    // Not level 3 (Mapped)
-    if ((int)label["PROCESSING_LEVEL_ID"] >= 3) {
```
Comment (acpaquette) [inline isis/src/mex/apps/hrsc2isis/hrsc2isis.cpp:74]: Can this check remain and we just set the processing level check to be `>= 4`?

File: isis/src/mex/apps/hrsc2isis/hrsc2isis.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
   <description> 
     <p>
       This program imports Mars Express HRSC files. This includes both the HRSC 
-      Stereo and the SRC instruments with procesing level of "1" or "2" (not "3"). 
+      Stereo and the SRC instruments.
```
Comment (acpaquette) [inline isis/src/mex/apps/hrsc2isis/hrsc2isis.xml]: Similarly with the above change, specify that we can't ingest anything >4?

**Review by acpaquette — COMMENTED** [review]
Overall looks good, small suggestions

File: isis/src/mex/apps/hrsc2isis/hrsc2isis.xml [hunk 3]
```diff
@@ -7,7 +7,7 @@
   <description> 
     <p>
       This program imports Mars Express HRSC files. This includes both the HRSC 
-      Stereo and the SRC instruments with procesing level of "1" or "2" (not "3"). 
+      Stereo and the SRC instruments  with procesing level of "1", "2", or "3" (not "4").
```
Comment (acpaquette) [inline isis/src/mex/apps/hrsc2isis/hrsc2isis.xml]: Extra space before "with"

**Review by acpaquette — COMMENTED** [review]
Overall looks great, one last grammar issue to fix

**Comment by acpaquette** [comment]
@amystamile-usgs Small changelog conflict that needs to be resolved

**Review by acpaquette — APPROVED** [review]

**Pull request merged**